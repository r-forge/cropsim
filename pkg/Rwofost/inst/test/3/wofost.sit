** File        : ..\RUNIO\WCCSI.SIT
** Generated by: c:\program files\wofost control center\exec\WCC.exe
** Date        : 15-7-2004 8:42:23
SOFILE = 'soil_5.sol'
IZT    = 0
IFUNRN = 0
IDRAIN = 0
SSMAX  = 0.000000
WAV    = 25.000000
ZTI    = 100.000000
DD     = 0.000000
RDMSOL = 110.000000
NOTINF = 0.000000
NBASE  = 60.000000
NREC   = 0.500000
PBASE  = 10.000000
PREC   = 0.100000
KBASE  = 60.000000
KREC   = 0.500000
SSI    = 0.000000
SMLIM  = 0.400000
