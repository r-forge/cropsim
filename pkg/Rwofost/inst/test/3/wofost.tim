** File        : ..\RUNIO\WCCTI.TIM
** Generated by: c:\program files\wofost control center\exec\WCC.exe
** Date        : 15-7-2004 8:42:23
RUNNAM  = 'WOFOST'
IWEATH  = 2
CLFILE  = 'PHL1.'
ISYR    = 1985
INYEAR  = 1
IRNDAT  = 3
RAFILE  = '<none>'
ISYRR   = -999
INYRG   = 1
CRFILE  = 'maiz.w41'
ISTCHO  = 0
IDEM    = 1
IDSOW   = 1
IDESOW  = 1
IDLSOW  = 0
IENCHO  = 1
IDAYEN  = 304
IDURMX  = 200
IBAL    = 3
PRDEL   = 1.000000
CRPNAM  = 'Maize'
SOLNAM  = 'EC soil_5'
CLMNAM  = 'Los Banos, Phillipines'
RAINAM  = '<none>'
ISDAY   = 1
