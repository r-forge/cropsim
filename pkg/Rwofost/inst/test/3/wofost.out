

 **WOFOST version 7.1.7, release September 2013
 RUNNAM -> WOFOST
 OUTPUT -> file: ../output/wofost.out
 RERUNS -> no reruns

 WEATHER-> name: Los Banos, Phillipines
           file: ../meteo/cabowe/phl1.   start year: 1985
 RAIN   -> belonging to weather station
 CROP   -> name: Maize,(Van Heemst, 1988)
           file: ../cropd/maiz.w41
 SOIL   -> name: soil_5
           file: ../soild/soil_5.sol
 START  -> fixed emergence date IDEM =  1

     start waterbalance = -99     sowing date = -99     emergence date =   1

 POTENTIAL CROP PRODUCTION
 =========================

 YEAR  DAY IDSEM DVS  TSUM   WLV    WST    WSO    TAGP   LAI  TRA   GASS   MRES    DMI

                     degrd kg/ha  kg/ha  kg/ha   kg/ha m2/m2   mm/d CH2O CH2O  kg/ha/d
  1985   1   0  0.00    0.     7.     5.     0.    12.  0.03 0.03    5.8    0.4    3.9
  1985   2   1  0.02   17.     9.     5.     0.    14.  0.03 0.04    7.3    0.4    4.9
  1985   3   2  0.04   33.    11.     7.     0.    17.  0.04 0.04    7.8    0.5    5.2
  1985   4   3  0.07   49.    13.     8.     0.    20.  0.04 0.05   11.2    0.6    7.6
  1985   5   4  0.09   65.    16.    10.     0.    25.  0.05 0.05   10.5    0.7    7.0
  1985   6   5  0.11   81.    18.    11.     0.    30.  0.06 0.06   11.4    0.8    7.6
  1985   7   6  0.13   97.    21.    13.     0.    34.  0.07 0.09   18.4    0.9   12.5
  1985   8   7  0.15  111.    26.    16.     0.    43.  0.09 0.12   26.0    1.1   17.8
  1985   9   8  0.17  126.    34.    21.     0.    54.  0.11 0.14   29.9    1.3   20.3
  1985  10   9  0.19  140.    42.    26.     0.    68.  0.14 0.18   41.3    1.8   28.1
  1985  11  10  0.21  155.    54.    33.     0.    86.  0.18 0.23   50.7    2.3   34.5
  1985  12  11  0.23  171.    68.    42.     0.   110.  0.22 0.29   61.0    3.0   41.3
  1985  13  12  0.25  186.    85.    52.     0.   138.  0.27 0.37   73.7    3.8   49.8
  1985  14  13  0.27  202.   107.    65.     0.   172.  0.34 0.38   66.1    4.3   44.0
  1985  15  14  0.29  217.   126.    77.     0.   203.  0.40 0.41   70.9    5.5   46.6
  1985  16  15  0.31  232.   146.    90.     0.   236.  0.46 0.44   71.8    6.1   46.8
  1985  17  16  0.33  248.   167.   102.     0.   269.  0.52 0.32   52.1    7.3   31.9
  1985  18  17  0.35  264.   181.   111.     0.   292.  0.56 0.60  111.0    9.0   72.5
  1985  19  18  0.38  282.   214.   131.     0.   345.  0.65 0.63   84.1   10.3   52.5
  1985  20  19  0.40  299.   238.   146.     0.   383.  0.72 0.89  147.7   11.3   97.0
  1985  21  20  0.42  316.   283.   173.     0.   456.  0.84 0.96  165.9   12.7  109.0
  1985  22  21  0.44  333.   333.   204.     0.   538.  0.98 0.95  173.0   14.6  112.6
  1985  23  22  0.47  349.   387.   237.     0.   624.  1.12 0.92  148.6   18.1   92.7
  1985  24  23  0.49  367.   431.   264.     0.   695.  1.23 1.48  265.8   20.6  174.2
  1985  25  24  0.51  384.   514.   317.     0.   830.  1.45 1.66  296.0   23.2  193.7
  1985  26  25  0.54  401.   604.   378.     0.   983.  1.68 1.45  273.3   27.1  174.7
  1985  27  26  0.56  418.   684.   437.     0.  1122.  1.87 1.54  278.9   31.9  175.1
  1985  28  27  0.58  436.   762.   500.     0.  1262.  2.06 1.76  294.8   36.1  183.3
  1985  29  28  0.60  453.   843.   568.     0.  1411.  2.26 2.00  334.9   39.2  209.4
  1985  30  29  0.63  471.   932.   651.     0.  1583.  2.47 2.12  345.3   43.4  213.5
  1985  31  30  0.65  488.  1020.   739.     0.  1759.  2.67 2.16  369.0   47.0  227.6
  1985  32  31  0.67  505.  1112.   838.     0.  1950.  2.88 2.13  358.9   54.0  215.3
  1985  33  32  0.70  523.  1196.   935.     0.  2132.  3.07 2.61  453.9   60.8  277.3
  1985  34  33  0.72  541.  1301.  1067.     0.  2368.  3.29 2.91  488.8   65.9  298.1
  1985  35  34  0.75  559.  1411.  1215.     0.  2625.  3.53 3.05  516.1   75.3  310.4
  1985  36  35  0.77  577.  1520.  1375.     0.  2896.  3.76 2.99  498.7   83.2  292.3
  1985  37  36  0.79  596.  1619.  1533.     0.  3153.  3.96 3.20  550.6   93.5  321.3
  1985  38  37  0.82  616.  1724.  1715.     0.  3438.  4.16 3.75  584.1   95.4  343.1
  1985  39  38  0.85  634.  1830.  1917.     0.  3747.  4.37 2.52  427.9   96.5  232.4
  1985  40  39  0.87  652.  1898.  2059.     0.  3958.  4.50 3.56  584.1  105.8  335.1
  1985  41  40  0.89  670.  1992.  2273.     0.  4265.  4.67 3.76  596.6  107.4  342.4
  1985  42  41  0.92  688.  2082.  2499.     0.  4581.  4.84 4.04  607.0  120.1  340.9
  1985  43  42  0.94  706.  2167.  2721.    12.  4900.  4.98 3.60  588.9  126.9  323.8
  1985  44  43  0.97  725.  2242.  2923.    39.  5205.  5.11 3.87  610.9  130.5  337.1
  1985  45  44  0.99  743.  2315.  3125.    86.  5525.  5.23 3.71  591.8  135.4  320.6
  1985  46  45  1.00  761.  2379.  3309.   146.  5833.  5.34 3.67  580.6  154.5  299.5
  1985  47  46  1.02  780.  2437.  3477.   209.  6122.  5.43 3.91  627.5  153.8  333.3
  1985  48  47  1.04  799.  2494.  3656.   295.  6446.  5.52 3.92  615.7  150.2  327.9
  1985  49  48  1.06  816.  2544.  3823.   395.  6767.  5.60 3.75  588.7  168.4  296.3
  1985  50  49  1.09  835.  2584.  3968.   500.  7059.  5.65 3.49  583.1  160.8  298.1
  1985  51  50  1.11  853.  2618.  4105.   620.  7356.  5.70 3.03  492.1  172.3  225.9
  1985  52  51  1.13  871.  2639.  4204.   722.  7582.  5.72 4.08  657.2  177.0  339.7
  1985  53  52  1.15  890.  2666.  4342.   891.  7922.  5.75 4.01  638.5  178.9  325.6
  1985  54  53  1.17  908.  2685.  4464.  1068.  8247.  5.78 3.09  502.7  175.4  232.1
  1985  55  54  1.19  926.  2693.  4545.  1205.  8480.  5.78 3.26  493.5  193.1  213.3
  1985  56  55  1.21  945.  2696.  4613.  1340.  8693.  5.77 3.90  625.6  195.3  306.0
  1985  57  56  1.24  963.  2698.  4702.  1549.  8999.  5.77 2.69  406.7  194.4  151.2
  1985  58  57  1.26  982.  2694.  4741.  1659.  9150.  5.75 4.07  633.1  216.1  297.3
  1985  59  58  1.28 1001.  2687.  4808.  1889.  9447.  5.73 3.97  605.3  205.1  285.9
  1985  60  59  1.30 1020.  2681.  4859.  2124.  9733.  5.71 3.42  533.7  208.4  232.8
  1985  61  60  1.32 1039.  2676.  4891.  2325.  9966.  5.69 4.32  657.3  209.6  321.0
  1985  62  61  1.35 1058.  2670.  4921.  2616. 10287.  5.68 4.36  658.8  208.8  323.3
  1985  63  62  1.37 1076.  2665.  4936.  2924. 10610.  5.66 4.33  649.4  213.0  314.1
  1985  64  63  1.39 1094.  2660.  4938.  3236. 10925.  5.64 4.11  636.4  214.1  304.0
  1985  65  64  1.41 1113.  2656.  4938.  3540. 11229.  5.63 4.37  631.9  214.2  300.7
  1985  66  65  1.43 1131.  2651.  4938.  3841. 11529.  5.62 4.60  647.6  237.0  295.6
  1985  67  66  1.45 1150.  2647.  4938.  4136. 11825.  5.60 3.16  517.0  222.7  211.9
  1985  68  67  1.47 1168.  2643.  4938.  4348. 12037.  5.59 3.15  505.4  217.2  207.5
  1985  69  68  1.49 1186.  2639.  4938.  4556. 12244.  5.58 2.95  455.5  218.5  170.7
  1985  70  69  1.52 1203.  2635.  4938.  4726. 12415.  5.57 3.00  468.9  217.1  181.3
  1985  71  70  1.54 1221.  2632.  4840.  4908. 12596.  5.56 3.99  616.0  224.8  281.7
  1985  72  71  1.56 1239.  2628.  4743.  5189. 12878.  5.55 3.60  572.4  231.5  245.4
  1985  73  72  1.58 1257.  2625.  4648.  5435. 13123.  5.54 4.28  630.2  243.9  278.2
  1985  74  73  1.60 1276.  2622.  4555.  5713. 13401.  5.53 3.79  563.4  227.1  242.1
  1985  75  74  1.62 1294.  2619.  4464.  5955. 13644.  5.52 3.78  581.1  236.0  248.5
  1985  76  75  1.64 1312.  2617.  4375.  6204. 13892.  5.51 3.01  459.6  235.3  161.4
  1985  77  76  1.66 1331.  2604.  4287.  6365. 14053.  5.48 4.77  632.5  260.8  267.6
  1985  78  77  1.69 1351.  2583.  4201.  6633. 14321.  5.42 3.30  522.3  239.0  204.0
  1985  79  78  1.71 1369.  2569.  4117.  6837. 14525.  5.38 4.33  602.2  260.9  245.8
  1985  80  79  1.73 1389.  2536.  4035.  7082. 14771.  5.28 4.35  599.4  259.1  245.0
  1985  81  80  1.75 1408.  2512.  3954.  7327. 15016.  5.22 4.21  585.5  255.1  237.8
  1985  82  81  1.78 1428.  2467.  3875.  7565. 15254.  5.09 4.37  558.9  235.2  233.0
  1985  83  82  1.80 1446.  2417.  3798.  7798. 15487.  4.96 4.37  528.9  256.8  195.9
  1985  84  83  1.82 1466.  2363.  3722.  7994. 15683.  4.81 3.42  447.8  250.4  142.1
  1985  85  84  1.84 1485.  2236.  3647.  8136. 15825.  4.49 4.25  461.1  253.1  149.8
  1985  86  85  1.87 1505.  2146.  3574.  8286. 15975.  4.26 3.73  410.6  249.2  116.2
  1985  87  86  1.89 1524.  2066.  3503.  8402. 16091.  4.06 3.86  368.5  246.3   88.0
  1985  88  87  1.91 1544.  1988.  3433.  8490. 16179.  3.87 4.05  307.2  232.4   53.8
  1985  89  88  1.93 1563.  1908.  3364.  8544. 16233.  3.68 4.07  248.2  231.3   12.1
  1985  90  89  1.96 1582.  1818.  3297.  8556. 16245.  3.47 3.80  182.2  182.2    0.0
  1985  91  90  1.98 1601.  1730.  3231.  8556. 16245.  3.26 2.28   85.7   85.7    0.0
  1985  92  91  2.00 1619.  1638.  3166.  8556. 16245.  3.05 3.26    2.5    2.5    0.0
  1985  93  92  2.02 1639.  1554.  3103.  8556. 16245.  2.87 3.19    0.0    0.0    0.0

 SUMMARY:
 HALT  ANTH TWRT    TWLV    TWST    TWSO    TAGP HINDEX TRANSP TRC  GASST  MREST
  93  45   1006.   2750.   4938.   8556.  16245.   0.53  24.6 151. 35741. 11511.
1


 **WOFOST version 7.1.7, release September 2013
 RUNNAM -> WOFOST
 OUTPUT -> file: ../output/wofost.out
 RERUNS -> no reruns

 WEATHER-> name: Los Banos, Phillipines
           file: ../meteo/cabowe/phl1.   start year: 1985
 RAIN   -> belonging to weather station
 CROP   -> name: Maize,(Van Heemst, 1988)
           file: ../cropd/maiz.w41
 SOIL   -> name: soil_5
           file: ../soild/soil_5.sol
 START  -> fixed emergence date IDEM =  1

     start waterbalance =   1 (1985)  sowing date = -99  emergence date =   1


 WATER LIMITED CROP PRODUCTION WITHOUT GROUNDWATER
 =================================================
 fixed fraction       RDMso=110.   NOTinf=0.00
 SM0=0.400 SMFC=0.263 SMW=0.100 SMLIM=0.400    RDM= 70. WAV= 25.0 SSmax= 0.0

 YEAR  DAY   WLV   WST    WSO   TAGP   LAI  RD    SM RESRV AVAIL RAIN TRA EVA wet dry

           kg/ha kg/ha  kg/ha  kg/ha m2/m2  cm vol.fr  cm    cm    mm mm/d mm/d  days
  1985   1     7.     5.     0.    12.  0.03  10. 0.400 21.0  3.0    0. 0.03 1.10  0  0
  1985   2     9.     5.     0.    14.  0.03  11. 0.264 11.4  1.8    0. 0.04 0.94  0  0
  1985   3    11.     7.     0.    17.  0.04  12. 0.256 11.3  1.9    0. 0.04 0.65  0  0
  1985   4    13.     8.     0.    20.  0.04  14. 0.255 11.3  2.1    1. 0.05 1.16  0  0
  1985   5    16.    10.     0.    25.  0.05  15. 0.248 11.2  2.2    1. 0.05 0.50  0  0
  1985   6    18.    11.     0.    30.  0.06  16. 0.248 11.2  2.4    1. 0.06 0.85  0  0
  1985   7    21.    13.     0.    34.  0.07  17. 0.261 11.4  2.8    4. 0.09 2.89  0  0
  1985   8    26.    16.     0.    43.  0.09  18. 0.245 11.1  2.7    4. 0.12 0.56  0  0
  1985   9    34.    21.     0.    54.  0.11  20. 0.243 11.0  2.8    4. 0.14 0.48  0  0
  1985  10    42.    26.     0.    68.  0.14  21. 0.241 10.9  2.9    4. 0.18 0.48  0  0
  1985  11    54.    33.     0.    86.  0.18  22. 0.239 10.9  3.1    4. 0.23 0.45  0  0
  1985  12    68.    42.     0.   110.  0.22  23. 0.237 10.8  3.2    4. 0.29 0.43  0  0
  1985  13    85.    52.     0.   138.  0.27  24. 0.236 10.7  3.3    4. 0.37 0.41  0  0
  1985  14   107.    65.     0.   172.  0.34  26. 0.234 10.7  3.4    4. 0.38 0.31  0  0
  1985  15   126.    77.     0.   203.  0.40  27. 0.233 10.6  3.6    4. 0.41 0.38  0  0
  1985  16   146.    90.     0.   236.  0.46  28. 0.234 10.6  3.7    5. 0.44 0.95  0  0
  1985  17   167.   102.     0.   269.  0.52  29. 0.230 10.5  3.8    5. 0.32 0.15  0  0
  1985  18   181.   111.     0.   292.  0.56  30. 0.230 10.4  4.0    5. 0.60 0.26  0  0
  1985  19   214.   131.     0.   345.  0.65  32. 0.229 10.3  4.1    5. 0.63 0.21  0  0
  1985  20   238.   146.     0.   383.  0.72  33. 0.227 10.2  4.2    5. 0.89 0.27  0  0
  1985  21   283.   173.     0.   456.  0.84  34. 0.225 10.1  4.3    5. 0.96 0.24  0  0
  1985  22   333.   204.     0.   538.  0.98  35. 0.223 10.0  4.3    5. 0.95 0.19  0  0
  1985  23   387.   237.     0.   624.  1.12  36. 0.222  9.9  4.4    5. 0.92 0.25  0  0
  1985  24   431.   264.     0.   695.  1.23  38. 0.220  9.8  4.5    5. 1.48 0.22  0  0
  1985  25   514.   317.     0.   830.  1.45  39. 0.217  9.6  4.5    5. 1.66 0.19  0  0
  1985  26   604.   378.     0.   983.  1.68  40. 0.216  9.5  4.6    6. 1.45 1.14  0  0
  1985  27   684.   437.     0.  1122.  1.87  41. 0.221  9.7  5.0   10. 1.54 1.23  0  0
  1985  28   762.   500.     0.  1262.  2.06  42. 0.216  9.4  4.9   10. 1.76 0.31  0  0
  1985  29   843.   568.     0.  1411.  2.26  44. 0.213  9.2  4.9   10. 2.00 0.11  0  0
  1985  30   932.   651.     0.  1583.  2.47  45. 0.209  9.0  4.9   10. 2.12 0.10  0  0
  1985  31  1020.   739.     0.  1759.  2.67  46. 0.210  9.0  5.1   12. 2.16 0.98  0  0
  1985  32  1112.   838.     0.  1950.  2.88  47. 0.207  8.7  5.0   13. 2.13 0.77  0  0
  1985  33  1196.   935.     0.  2132.  3.07  48. 0.202  8.5  4.9   13. 2.61 0.08  0  0
  1985  34  1301.  1067.     0.  2368.  3.29  50. 0.205  8.5  5.2   16. 2.91 0.92  0  0
  1985  35  1411.  1215.     0.  2625.  3.53  51. 0.198  8.1  5.0   16. 3.05 0.07  0  0
  1985  36  1520.  1375.     0.  2896.  3.76  52. 0.195  7.9  4.9   17. 2.99 0.66  0  0
  1985  37  1619.  1533.     0.  3153.  3.96  53. 0.190  7.5  4.8   17. 3.20 0.06  0  0
  1985  38  1724.  1715.     0.  3438.  4.16  54. 0.185  7.2  4.6   17. 3.75 0.06  0  0
  1985  39  1830.  1917.     0.  3747.  4.37  56. 0.180  6.8  4.5   17. 2.52 0.03  0  0
  1985  40  1898.  2059.     0.  3958.  4.50  57. 0.178  6.6  4.4   17. 3.56 0.05  0  0
  1985  41  1992.  2273.     0.  4265.  4.67  58. 0.173  6.2  4.2   17. 3.76 0.04  0  0
  1985  42  2082.  2499.     0.  4581.  4.84  59. 0.168  5.8  4.1   17. 4.04 0.04  0  0
  1985  43  2167.  2721.    12.  4900.  4.98  60. 0.164  5.4  3.8   17. 3.60 0.04  0  0
  1985  44  2242.  2923.    39.  5205.  5.11  62. 0.160  5.0  3.7   17. 3.87 0.03  0  0
  1985  45  2315.  3125.    86.  5525.  5.23  63. 0.155  4.7  3.5   17. 3.71 0.03  0  0
  1985  46  2379.  3309.   146.  5833.  5.34  64. 0.152  4.3  3.3   17. 3.65 0.03  0  0
  1985  47  2436.  3476.   208.  6121.  5.43  65. 0.148  3.9  3.1   17. 3.41 0.03  0  1
  1985  48  2476.  3625.   280.  6390.  5.48  66. 0.145  3.6  3.0   17. 3.20 0.03  0  1
  1985  49  2502.  3752.   356.  6633.  5.49  68. 0.142  3.2  2.8   17. 3.01 0.03  0  1
  1985  50  2519.  3858.   433.  6847.  5.50  69. 0.140  2.9  2.7   17. 2.85 0.02  0  1
  1985  51  2536.  3962.   524.  7073.  5.50  70. 0.138  2.6  2.6   17. 2.77 0.02  0  1
  1985  52  2552.  4050.   615.  7274.  5.52  70. 0.134  2.4  2.4   17. 2.42 0.03  0  1
  1985  53  2537.  4113.   693.  7430.  5.45  70. 0.130  2.1  2.1   17. 2.16 0.03  0  1
  1985  54  2512.  4160.   761.  7556.  5.36  70. 0.127  1.9  1.9   17. 1.98 0.02  0  1
  1985  55  2493.  4200.   828.  7670.  5.30  70. 0.124  1.7  1.7   17. 1.75 0.02  0  1
  1985  56  2462.  4220.   868.  7733.  5.20  70. 0.123  1.6  1.6   18. 1.60 0.44  0  1
  1985  57  2421.  4237.   908.  7792.  5.08  70. 0.121  1.5  1.5   19. 1.60 0.30  0  1
  1985  58  2393.  4251.   946.  7844.  5.01  70. 0.119  1.3  1.3   19. 1.30 0.03  0  1
  1985  59  2345.  4253.   955.  7855.  4.88  70. 0.117  1.2  1.2   19. 1.16 0.03  0  1
  1985  60  2296.  4254.   958.  7859.  4.75  70. 0.115  1.1  1.1   19. 1.04 0.03  0  1
  1985  61  2249.  4254.   958.  7859.  4.62  70. 0.114  1.0  1.0   19. 0.98 0.54  0  1
  1985  62  2198.  4254.   958.  7859.  4.49  70. 0.112  0.8  0.8   19. 0.82 0.04  0  1
  1985  63  2145.  4254.   958.  7859.  4.35  70. 0.111  0.8  0.8   19. 0.73 0.04  0  1
  1985  64  2092.  4254.   958.  7859.  4.22  70. 0.110  0.7  0.7   19. 0.64 0.05  0  1
  1985  65  2040.  4254.   958.  7859.  4.09  70. 0.109  0.6  0.6   19. 0.58 0.05  0  1
  1985  66  1988.  4254.   958.  7859.  3.96  70. 0.108  0.5  0.5   19. 0.51 0.06  0  1
  1985  67  1936.  4254.   958.  7859.  3.83  70. 0.112  0.8  0.8   23. 0.77 0.67  0  1
  1985  68  1893.  4254.   958.  7859.  3.72  70. 0.113  0.9  0.9   25. 0.85 0.69  0  1
  1985  69  1854.  4254.   958.  7859.  3.63  70. 0.118  1.3  1.3   30. 1.20 0.67  0  1
  1985  70  1824.  4254.   989.  7890.  3.56  70. 0.123  1.6  1.6   35. 1.47 0.70  0  1
  1985  71  1800.  4169.  1054.  7955.  3.50  70. 0.120  1.4  1.4   35. 1.26 0.36  0  1
  1985  72  1766.  4085.  1090.  7991.  3.42  70. 0.118  1.3  1.3   35. 1.10 0.05  0  1
  1985  73  1732.  4004.  1112.  8013.  3.34  70. 0.116  1.2  1.2   36. 1.00 0.17  0  1
  1985  74  1694.  3924.  1112.  8013.  3.25  70. 0.115  1.0  1.0   36. 0.89 0.06  0  1
  1985  75  1658.  3845.  1112.  8013.  3.17  70. 0.119  1.3  1.3   39. 1.12 1.08  0  1
  1985  76  1626.  3768.  1136.  8037.  3.09  70. 0.119  1.3  1.3   41. 1.12 0.87  0  1
  1985  77  1599.  3693.  1162.  8063.  3.03  70. 0.118  1.2  1.2   43. 1.05 1.44  0  1
  1985  78  1564.  3619.  1162.  8063.  2.96  70. 0.114  1.0  1.0   43. 0.83 0.16  0  1
  1985  79  1532.  3547.  1162.  8063.  2.88  70. 0.121  1.5  1.5   49. 1.22 1.41  0  1
  1985  80  1502.  3476.  1181.  8082.  2.82  70. 0.118  1.2  1.2   49. 0.99 0.08  0  1
  1985  81  1470.  3406.  1181.  8082.  2.74  70. 0.116  1.1  1.1   49. 0.89 0.08  0  1
  1985  82  1438.  3338.  1181.  8082.  2.67  70. 0.115  1.0  1.0   49. 0.81 0.09  0  1
  1985  83  1405.  3271.  1181.  8082.  2.60  70. 0.113  0.9  0.9   49. 0.73 0.09  0  1
  1985  84  1372.  3206.  1181.  8082.  2.53  70. 0.112  0.9  0.9   49. 0.65 0.07  0  1
  1985  85  1341.  3142.  1181.  8082.  2.46  70. 0.111  0.8  0.8   49. 0.59 0.10  0  1
  1985  86  1308.  3079.  1181.  8082.  2.39  70. 0.110  0.7  0.7   49. 0.53 0.09  0  1
  1985  87  1276.  3017.  1181.  8082.  2.32  70. 0.109  0.7  0.7   49. 0.47 0.09  0  1
  1985  88  1244.  2957.  1181.  8082.  2.26  70. 0.109  0.6  0.6   49. 0.43 0.10  0  1
  1985  89  1211.  2898.  1181.  8082.  2.19  70. 0.108  0.5  0.5   49. 0.39 0.11  0  1
  1985  90  1180.  2840.  1181.  8082.  2.12  70. 0.107  0.5  0.5   49. 0.34 0.11  0  1
  1985  91  1148.  2783.  1181.  8082.  2.06  70. 0.107  0.5  0.5   49. 0.33 0.26  0  1
  1985  92  1120.  2727.  1181.  8082.  2.00  70. 0.107  0.5  0.5   50. 0.32 0.90  0  1
  1985  93  1091.  2673.  1181.  8082.  1.94  70. 0.105  0.4  0.4   50. 0.24 0.10  0  1

 SUMMARY :                                                        stress days
 HALT ANTH TWRT   TWLV   TWST   TWSO   TAGP HINDEX TRC  GASST  MREST wet dry
  93  45  1002.  2647.  4254.  1181.  8082.  0.15 157.9 20320.  7474.   0  47

       WATER BALANCE WHOLE SYSTEM (1 DIMENS. COLUMN ; cm)
 init max root zone    28.0  final max root zone    7.4    change: -20.6
  init surf storage     0.0  final surf storage     0.0    change:   0.0
         irrigation     0.0  evap water surface     0.0
           rainfall     5.0  evap soil surface      3.4
                             transpiration         12.8  to atmos:  16.1
                             surface runoff         0.0
                             lost to deep soil      9.5
  TOTAL INIT + IN      33.0  TOTAL FINAL + OUT     33.0  checksum:   0.0

               WATER BALANCE ROOT ZONE
 initial water stock    4.0  final water stock      7.4
        infiltration    5.0  evap soil surface      3.4
 added by root growth  15.8      transpiration     12.8
                                   percolation      1.3
 TOTAL INIT + IN       24.8  FINAL + OUT           24.7  checksum:   0.0
